scripts/evaluate_ner.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+
+import spacy
+import importlib
+
+from scispacy.data_util import read_full_med_mentions, read_ner_from_tsv
+from scispacy.train_utils import evaluate_ner
+
+
+def main(model_path: str, dataset: str, output_path: str, code: Optional[str]):
+    if code is not None:
```

Comment:
Does this happen automatically inside the created models? Presumably, right? Is it just because we haven't packaged them at this point that we need this additional code injection?

Reply:
Right, this is exactly what all the spacy CLI commands that accept `--code` do. Also we're actually going to be able to get ride of the `--code` bit (except for training) after rc3 is released, because the tokenizer callback will be in a different place such that it only needs to happen when our base model is made. And, I wanted to eval the models before packaging separately from after packaging because of that one time we had a bug that caused a difference